packages/@coorpacademy-components/src/molecule/progress-wrapper/index.js

```diff
@@ -88,32 +100,37 @@ const DetailSection = ({index, type, isLocked, badgeUrl, onDownload, stars}, con
 };
 
 const ProgressWrapper = (
-  {completedCourses, completedModules, title, subtitle, progression, sections},
+  {completedModules, mandatoryModules, title, subtitle, progression, sections},
```

Comment:
still this course/modules/external/chapter situation 😅

Reply:
completedContentModuleGranularity ?
complex naming... I don't know